MonoGame.Framework/Point.cs

```diff
@@ -77,6 +77,37 @@ public Point(int x, int y)
             this.Y = y;
         }
 
+        /// <summary>
+        /// Creates a new instance of <see cref="Point"/> struct, with the specified position.
+        /// </summary>
+        /// <param name="value">The x and y coordinates in 2d-space.</param>
+        public Point(int value)
+        {
+            this.X = value;
+            this.Y = value;
+        }
+
+        /// <summary>
+        /// Creates a new instance of <see cref="Point"/> struct, with the specified position.
+        /// </summary>
+        /// <param name="x">The x coordinate in 2d-space.</param>
+        /// <param name="y">The y coordinate in 2d-space.</param>
+        public Point(float x, float y)
```

Comment:
Ok, so if I delete float overload's from Rectangle and Point, than my PR can be merged ?